tests/integration/ivaDefects.intTest.ts

```diff
@@ -19,61 +18,34 @@ describe("Defects Service", () => {
             (first: { imNumber: number }, second: { imNumber: number }) =>
```

Comment:
Why is this sorting needed?

Reply:
tests still pass without sorting. Removed.